README.md

```diff
@@ -3,29 +3,48 @@
 [![NPM Package](https://img.shields.io/npm/v/@openzeppelin/test-helpers.svg)](https://www.npmjs.org/package/@openzeppelin/test-helpers)
 [![Build Status](https://travis-ci.com/OpenZeppelin/openzeppelin-test-helpers.svg?branch=master)](https://travis-ci.com/OpenZeppelin/openzeppelin-test-helpers)
 
-**JavaScript testing helpers for Ethereum smart contract development.** With support for Truffle and plain web3.js workflows.
+*Assertion library for Ethereum smart contract testing.* Make sure your contracts behave as expected!
 
-## Installation
+ * Check that [transactions revert](docs/modules/ROOT/pages/api.adoc#expect-revert) for the correct reason
+ * Verify [events](docs/modules/ROOT/pages/api.adoc#expect-event) where emitted with the right values
+ * Track [balance changes](docs/modules/ROOT/pages/api.adoc#balance) elegantly
+ * Handle [very large numbers](docs/modules/ROOT/pages/api.adoc#bn)
+ * Simulate the [passing of time](docs/modules/ROOT/pages/api.adoc#time)
+
+Test Helpers integrates seamlessly with [OpenZeppelin Test Environment](https://github.com/OpenZeppelin/openzeppelin-test-environment), but it also supports both Truffle tests and regular web3 workflows.
+
+## Overview
+
+### Installation
 
 ```bash
 npm install --save-dev @openzeppelin/test-helpers
 ```
 
-## Usage
+### Usage
```

Comment:
Should we add a note saying that the snippet uses `test-environment` but that it works the same way with Truffle or something?